tests/tooltips/tooltips_test.js

```diff
@@ -22,4 +22,32 @@ describe("General Tooltip Tests", function(){
         TTE.remove();
         expect($('.tooltip').length).to.be.equal(0);
     });
-});
\ No newline at end of file
+
+    it("ACE anti-undo hack still works", function() {
```

Comment:
Eventually the color picker will use this hack too...? That's why I put it here instead of there.